cmd/minikube/cmd/start.go

```diff
@@ -371,6 +372,20 @@ func provisionWithDriver(cmd *cobra.Command, ds registry.DriverState, existing *
 	}, nil
 }
 
+func virtualBoxMacOS13PlusWarning(driverName string) {
+	if driverName != "virtualbox" || !detect.MacOS13Plus() {
+		return
+	}
+	driver := "hyperkit"
+	if runtime.GOARCH == "arm64" {
+		driver = "qemu"
+	}
+	out.WarningT(`Due to changes in macOS 13+ minikube doesn't currently support VirtualBox. We recommend using the {{.driver}} driver: https://minikube.sigs.k8s.io/docs/drivers/{{.driver}}/
```

Comment:
instead of we...

let say something like
"You could use an alternative driver such as Qemu or Docker"

Reply:
Updated message